hs_core/templates/pages/genericresource.html

```diff
@@ -830,8 +831,22 @@ <h3>Resource Specific</h3>
       const CURRENT_USER_ID = {{ request.user.id }};
       const SHORT_ID = "{{ cm.short_id }}";
       const STATIC_URL = "{{ STATIC_URL }}";
-      let spatial_coverage_type = "{{ spatial_coverage.type }}";
+      {% if show_manage_access %}
+          const USERS_JSON = {{ users_json|safe }};
+          const QUOTA_HOLDER_PK = {{ quota_holder.pk  }};
+          const CURRENT_USER_PK = {{ current_user.pk }};
+          const GROUP_IMAGE_DEFAULT_URL = "{{ STATIC_URL }}" + "img/home-page/step4.png";
+          const CAN_CHANGE_RESOURCE_FLAGS = "{{ can_change_resource_flags }}" === "True";
+          const CAN_BE_PUBLIC_OR_DISCOVERABLE = "{{ cm.can_be_public_or_discoverable }}" === "True";
+          const RESOURCE_ACCESS = {
+              isPublic: "{{ cm.raccess.public }}" === "True",
+              isDiscoverable: "{{ cm.raccess.discoverable }}" === "True",
+              isShareable: "{{ cm.raccess.shareable }}" === "True"
+          };
+      {% endif %}
+      var spatial_coverage_type = "{{ spatial_coverage.type }}";
```

Comment:
Can conditional logic be moved to Vue or is that too complicated or even impossible here?

Reply:
I don't think it would be cleaner than this. Adding a JavaScript conditional here would imply still using a Django template tag to read `show_manage_access`  and then using it as a js conditional to declare the global constants inside the `If` statement scope (which is not recommended).